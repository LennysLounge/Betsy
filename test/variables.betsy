var number 12
var other_number 144

set other_number 69
set number other_number

print number
print other_number