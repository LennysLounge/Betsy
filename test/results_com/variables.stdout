
Program output:
69
69
